### PR TITLE
Adding login module test for custom user

### PR DESCRIPTION
**Pull request by charvimehradu**
Pull Request for Issue # .

Adding login module test for custom user.

### CODE REVIEW

**Comment by laoneo** [comment]
Thank you very much for the pr. @Hackwar is right, this is a test for com_users. There is already a login module test, please add your code there https://github.com/joomla/joomla-cms/blob/4.4-dev/tests/System/integration/site/modules/mod_login/Default.cy.js. So we test then if the default user name works and from a custom user.

**Comment by charvimehradu** [comment]
Thank you! @Hackwar @laoneo I think I may have misunderstood your instructions the last time and I don't want to make the same mistake again. Could you please elaborate what I am supposed to do now?

**Comment by laoneo** [comment]
1. Add your test to the file I mentioned before
2. Write a similar test for the com_users login view

File: tests/System/integration/site/modules/mod_login/Default.cy.js [hunk 1]
```diff
@@ -4,3 +4,21 @@ describe('Test that the login module', () => {
...
+      username: 'test',
+      email: 'test@example.com',
+      password: '098f6bcd4621d373cade4e832627b4f6',
+    }).then(() => {
+      cy.visit('http://localhost/joomla-cms/');
+      cy.get('#modlgn-username-16').type('test');
+      cy.get('#modlgn-passwd-16').type('test');
+      cy.get('input[name="remember"]').check();
+      cy.get('.mod-login__submit > .btn').click();
+      cy.get('.alert-wrapper').should('contain.text', 'You have been logged in.');
```
Comment (laoneo) [inline tests/System/integration/site/modules/mod_login/Default.cy.js:20]:
```suggestion

       cy.get('.alert-wrapper').should('contain.text', 'You have been logged in.');
```

File: tests/System/integration/site/modules/mod_login/Default.cy.js [hunk 2]
```diff
@@ -4,3 +4,22 @@ describe('Test that the login module', () => {
...
+      email: 'test@example.com',
+      password: '098f6bcd4621d373cade4e832627b4f6',
+    }).then(() => {
+      cy.visit('http://localhost/joomla-cms/');
+      cy.get('#modlgn-username-16').type('test');
+      cy.get('#modlgn-passwd-16').type('test');
+      cy.get('input[name="remember"]').check();
+      cy.get('.mod-login__submit > .btn').click();
+
+       cy.get('.alert-wrapper').should('contain.text', 'You have been logged in.');
```
Comment (laoneo) [inline tests/System/integration/site/modules/mod_login/Default.cy.js]:
```suggestion
      cy.get('.alert-wrapper').should('contain.text', 'You have been logged in.');
```

File: tests/System/integration/site/modules/mod_login/Default.cy.js [hunk 3]
```diff
@@ -4,3 +4,22 @@ describe('Test that the login module', () => {
...
+
+describe('Test the Login module for test user ', () => {
+  it('can display the login module for a user', () => {
+    cy.db_createUser({
+      name: 'automated test user',
+      username: 'test',
+      email: 'test@example.com',
+      password: '098f6bcd4621d373cade4e832627b4f6',
+    }).then(() => {
+      cy.visit('http://localhost/joomla-cms/');
```
Comment (laoneo) [inline tests/System/integration/site/modules/mod_login/Default.cy.js]:
```suggestion
      cy.visit('/');
```

**Comment by charvimehradu** [comment]
> 2. Write a similar test for the com_users login view

Is it preferable to make a separate PR for this?

**Comment by laoneo** [comment]
Good idea, so we can merge this one early.

**Comment by laoneo** [comment]
Thanks!

**Pull request merged**